test/unit/convertV2.test.js

```diff
@@ -1170,6 +1171,33 @@ describe('The convert v2 Function', function() {
       });
   });
 
+  it('description test', function(done) {
```

Comment:
Test can be named better. e.g. `Should generate a collection with description for various Request Params` or `Should generate a collection with description for Query Params, Path variables and Headers`

Reply:
Done. Updated